packages/api/src/controllers/session.ts

```diff
@@ -187,4 +189,56 @@ const adminOnlyFields: (keyof DBSession)[] = ["deleted", "broadcasterHost"];
 
 const privateFields: (keyof DBSession)[] = ["recordObjectStoreId", "version"];
 
+export async function getRunningRecording(content: DBStream, req: Request) {
```

Comment:
Should we use this on `webhook/cannon.ts` as well? I think that's where the implementation came from right, not sure if it's different tho